lib/forwarder.js

```diff
@@ -12,15 +12,22 @@ import { removeConfig } from '../lib/config.js';
  * @param {Config} config
  */
 export async function forwardToDaemon(resolver, config) {
+  let eslintArgs = [...process.argv];
```

Comment:
How about the good old `process.argv.slice()`? 🤓

Reply:
changed. You don't like the spread operator?

Reply:
I like simple code. Spread is great, but somewhat overkill here.